pkg/machinehealthcheck/machinehealthcheck.go

```diff
@@ -20,7 +18,9 @@ var _ = Describe("[Feature:MachineHealthCheck] MachineHealthCheck", func() {
 
 	var machineSet *mapiv1beta1.MachineSet
 	var machineSetParams framework.MachineSetParams
+	var maxAllowedUnhealthy = 1
```

Comment:
can we name this consistently with api field, i.e `MaxUnhealthy`

Reply:
also should it be rather a constant?

Reply:
It should, but I can't pass a pointer to a constant.